docs/source/babs-init.rst

```diff
@@ -20,6 +20,11 @@ Command-Line Arguments
       :ref:`how-to-define-name-of-input-dataset` below for general guidelines
       and specific restrictions.
 
+   --keep-if-failed : @after
```

Comment:
Can you try this?

```suggestion
   --keep_if_failed, --keep-if-failed : @after
```

Reply:
Darn, sorry.

Reply:
That's okay, no worries! If you don't have other suggestions I might just add those words into CLI - won't matter a ton.